lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -210,17 +227,44 @@ def _get_saved_grade(self, subsection, course_structure, course):  # pylint: dis
                 )
                 subsection_grade = SubsectionGrade(subsection)
                 subsection_grade.load_from_data(model, course_structure, self._scores_client, self._submissions_scores)
+                log.warning(
+                    "Loaded saved grade for course with id {0}, version {1}, subsection {2}. grade: {3}".format(
+                        unicode(course.id),
+                        unicode(course.course_version),
```

Comment:
In addition to `course_version`, can you also add `course.subtree_edited_on` to get the timestamp?  Also since course_version is not available on old Mongo-backed courses.